tuf/scripts/repo.py

```diff
@@ -94,7 +94,8 @@
     
   --trust:
     Indicate trusted keys located in </path/to/pubkey> directory of a role.
-    
+  --expire:
```

Comment:
The `--threshold` parameter is missing here, no?

Reply:
Where are the password flags documented?

Reply:
the --threshold parameters is defined here https://github.com/theupdateframework/tuf/blob/develop/tuf/scripts/repo.py#L121 but is actually only use for delegate right now

the --pw field is documented here https://github.com/theupdateframework/tuf/blob/develop/tuf/scripts/repo.py#L107